### DISCUSSION TITLE
设置界面无法打开

### DISCUSSION THREAD

**Discussion by yours57** [shuax/MouseInc.Settings#409] [Q&A]
更新2.13.4之后，设置界面无法打开，下载离线包后也无法打开，之前可用
感谢开发者

**Comment by shuax** [comment]
webview2装了吗。  https://go.microsoft.com/fwlink/p/?LinkId=2124703

**Reply by yours57** [reply]
装了，显示我已安装   Microsoft Edge Webview2 Runtime.
之前版本用离线包是可以打开的

**Comment by shuax** [comment]
那就不知道了

**Reply by yours57** [reply]
我尝试部署了一下，打开后显示   设置通道连接失败，可能是MouseInc主程序未更新
还有比2.13.4更新的版本吗？请问这是什么原因？
或者还有其他什么方法吗？
谢谢！
